spec/models/info_request_spec.rb

```diff
@@ -562,6 +562,11 @@
       @ir = info_requests(:naughty_chicken_request)
     end
 
+    after do
+      Object.send(:remove_const, 'InfoRequest')
+      load 'app/models/info_request.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.